README.md

```diff
@@ -483,102 +490,128 @@ network_connections:
+          prefix: 26
+          gateway: 198.51.100.1
+          metric: 2
+        - network: 198.51.100.64
+          prefix: 26
+          gateway: 198.51.100.6
+          metric: 4
+      route_append_only: no
+      rule_append_only: yes
+```
 
-#### Invalid and Wrong Configuration
+### Invalid and Wrong Configuration
 
-The role will reject invalid configurations, so it is a good idea to test the role
+The `network` role rejects invalid configurations. It is recommended to test the role
 with `--check` first. There is no protection against wrong (but valid) configuration.
 Double-check your configuration before applying it.
 
-### Compatibility
 
-The role supports the same configuration scheme for both providers. That means, you can
-use the same playbook with NetworkManager and initscripts. Note however, that not every
-option is handled exactly the same by every provider. Do a test run first with `--check`.
+Compatibility
+-------------
 
-It is also not supported to create a configuration for one provider, and expect another
-provider to handle them. For example, creating proviles with `initscripts` provider
-and later enabling NetworkManager is not guaranteed to work automatically. Possibly
+The `network` role supports the same configuration scheme for both providers (`nm`
+and `initscripts`). That means, you can use the same playbook with NetworkManager
+and initscripts. However, note that not every option is handled exactly the same
+by every provider. Do a test run first with `--check`.
+
+It is not supported to create a configuration for one provider, and expect another
+provider to handle them. For example, creating profiles with the `initscripts` provider,
+and later enabling NetworkManager is not guaranteed to work automatically. Possibly,
 you have to adjust the configuration so that it can be used by another provider.
 
-For example what will work is to configure a RHEL6 host with initscripts and upgrade to
-RHEL7 while continuing to use initscripts on RHEL7. What is not guaranteed to work
-it to upgrade to RHEL7, disable initscripts and expect NetworkManager to take over
-the configuration automatically.
+For example, configuring a RHEL6 host with initscripts and upgrading to
+RHEL7 while continuing to use initscripts in RHEL7 is an acceptable scenario. What
+is not guaranteed is to upgrade to RHEL7, disable initscripts and expect NetworkManager
+to take over the configuration automatically.
 
 Depending on NetworkManager's configuration, connections may be stored as ifcfg files
-as well, but again it is not guaranteed that plain initscripts can handle these ifcfg files
+as well, but it is not guaranteed that plain initscripts can handle these ifcfg files
 after disabling the NetworkManager service.
+
+Limitations
+-----------
+
+Ansible usually works via the network, for example via SSH. There are some limitations to be considered:
+
+The `network` role does not  support how to bootstrap networking configuration. One
```

Comment:
This is less intelligible than the original version.

Reply:
What about this:

> As Ansible usually works via the network, for example via SSH, there are some limitations to be considered:
>
> The `network` role does not support bootstraping networking configuration. One…